3.7/README.md

```diff
@@ -94,6 +95,26 @@ docker run -it --rm --shm-size=1g \
   mediagis/nominatim:3.7
 ```
 
+## OpenStreetMap Data Extracts
+
+OSM data extracts (.pbf) can be imported with either PBF_URL or PBF_PATH variable.
+PBF_URL variable specifies the URL, the data is downloaded during initialization and removed from disk afterwards.
+PBF_PATH variable specifies the path inside the container to the mounted OSM extracts data. No .pbf file is removed after initialization and can used again.
+
+The replication update can be peformed only via HTTP.
+
+A sample of the docker configuration with PBF_PATH variable (point at the docker image that supports PBF_PATH) is:
+
+``` sh
+docker run -it --rm \
+  -e PBF_PATH=/nominatim/data/monaco-latest.osm.pbf \
+  -e REPLICATION_URL=https://download.geofabrik.de/europe/monaco-updates/ \
+  -p 8080:8080 \
+  -v C:\osm-maps\data:/nominatim/data \
+  --name nominatim \
+  mediagis/nominatim:3.7-pbf_path
```

Comment:
I think you need change this before we merge.

Reply:
fixed